### DISCUSSION TITLE
MacOS: Monterey ??????

### DISCUSSION THREAD

**Discussion by Uffynn** [ghostty-org/ghostty#3960] [Q&A] [answered]
How is this at least not running on Monterey? That literally make no sense. How do we get it to run on Monterey or even Big Sure as well. As both of the version of the MacOS literally run on 64-bit Intel and [ARM] without any issue.

**Comment by JakeWharton** [comment]
Only Apple-supported versions of macOS are currently supported by Ghostty.

https://endoflife.date/macos

**Reply by jcollie** [reply]
To expand a bit, there were APIs introduced in macOS 13 that Ghostty depends on. I'm not sure but I don't think that Ghostty ever ran on macOS 11.

**Reply by Uffynn** [reply]
Yeah, I saw that. But again, am not the only running a 2013/2014/2015 MacBook Pro. Millions of coders are daily. How do we make sure that we can support ghostty on Monterey & Big Sur? Since both version of the MacOS are 100% compatible with all the M-chip [arm]. 

https://en.wikipedia.org/wiki/MacOS_version_history

**Reply by 00-kat** [reply]
I don't think the fact that they are compatible with ARM chips is the issue here.  Rather, APIs (i.e. features that exist since macOS 13 that programs can use) that Ghostty depends on are not supported.

**Reply by Uffynn** [reply]
So what you are trying to say is that you think that its the 3rd parties that are used - from which we pull the info with APIs ?

**Reply by jcollie** [reply]
No, we are saying that Ghostty uses APIs that were introduced by Apple in macOS 13, which is why Ghostty does not work on macOS 12 or earlier. The type of CPU makes no difference.

**Reply by Uffynn** [reply]
Alright, that's what I thought. Is there anyone who might be interesting in forking this and making this run on Monterey & Big Sur?

**Comment by 00-kat** [comment, marked as answer]
Duplicate of #3696.

**Reply by Uffynn** [reply]
Not a duplicate. Am asking if there are devs out there who want to build a supportive version of Ghosty for Monterey & Big Sur. Obviously stating: "How do we get it to run on Monterey or even Big Sur as well" - wanting to start a discussion for those 2 OSs

**Reply by fightingdreamer** [reply]
@Uffynn you can build for Monterey using my [fork](https://github.com/fightingdreamer/ghostty/tree/macos-12).

**Comment by sujayxaradhya** [comment]
I'm a Big Sur user and there are millions of developers who want Ghostty to work on their older machines. How can they use Ghostty? Do they have to buy a new machine just to work with Ghostty? This is literally unfair by the Ghostty Team.

**Reply by pluiedev** [reply]
It's unfair to expect a volunteer team to support an operating system not even Apple supports anymore. 

You don't need to buy a new machine for Ghostty — you need to buy one just to have any amount of support and security fixes from Apple. Not our call to make. There are tools you could use to try to install newer macOS versions on your Mac but that is not something we'll endorse.

**Reply by infogrind** [reply]
You should consider Ghostty a gift: https://world.hey.com/dhh/the-open-source-gift-exchange-2171e0f0

**Reply by tristan957** [reply]
You can also try just not using macOS. It's a proprietary operating system that sucks the soul of every single living thing it touches. Linux is based and keeps old hardware working for much longer than vendors like Apple.

**Reply by sujayxaradhya** [reply]
> It's unfair to expect a volunteer team to support an operating system not even Apple supports anymore. 
> 
> You don't need to buy a new machine for Ghostty — you need to buy one just to have any amount of support and security fixes from Apple. Not our call to make. There are tools you could use to try to install newer macOS versions on your Mac but that is not something we'll endorse.

And it's also unfair for many terminals and new tools out there are somehow capable of supporting an operating system not even Apple supports anymore 😂, right. Just reply with 'List' and I will give you the endless list okay 😄.

If you were in the same boat, then the reply you @pluiedev quoted me above might be helpful for you not me.

**Reply by pluiedev** [reply]
If you wish to keep being entitled like this then you're free to not use Ghostty. However, please keep the disrespectfulness out of this thread.

**Reply by 00-kat** [reply]
https://dictionary.cambridge.org/dictionary/english/unfair

> Just reply with 'List' and I will give you the endless list okay 😄.

Sure, how many of your “endless” list are *volunteers* with **no obligation** to help develop the application?

> If you were in the same boat

You're free to continue using the terminal emulators you state support macOS versions not supported by Apple.  You're also free to fork Ghostty and take over the maintenance burden of supporting those macOS versions, as Mitchell has clearly stated in https://github.com/ghostty-org/ghostty/discussions/3696#discussioncomment-11703049.

> new tools

Conversely, would there really be *millions of developers* who would intentionally use *new* tools on an antiquated operating system?
